src/formatters/searchfox.js

```diff
@@ -0,0 +1,39 @@
+const searchfoxMap = {
+  fenix: "fenix",
+
+  "firefox-android-beta": "fenix",
+
+  "firefox-android-release": "fenix",
+
+  "firefox-desktop": "https://searchfox.org/mozilla-central/source/",
+
+  "firefox-focus-ios": "focus-ios",
+
+  "firefox-for-echo-show": "firefox-echo-show",
+
+  "firefox-for-fire-tv": "firefox-tv",
+
+  "firefox-ios-beta": "firefox-ios",
+
+  "firefox-ios-dev": "firefox-ios",
+
+  "firefox-ios-release": "firefox-ios",
+
+  "firefox-reality": "FirefoxReality",
+
+  "lockwise-android": "lockbox-android",
+
+  "reference-browser": "reference-browser",
+};
```

Comment:
I wonder if (longer term -- maybe as a follow-on to this work) this should be in probe-scraper's repositories.yaml instead.

Reply:
👍